mex/common/cli.py

```diff
@@ -122,21 +122,21 @@ def callback(
 
     # print .env-style settings if echo-settings flag is set, then exit.
     if cli_settings.get("echo_settings"):
-        click.secho(settings.env_text())
+        logger.info(click.style(settings.env_text()))
```

Comment:
`mex.common.logging.echo` just adds a coloured timestamp at the beginning of each line. but for printing the settings, we just want them without settings. that's why i use logging directly here.

and i threw out `click.echo` and `click.secho` entirely, because i had the feeling it doesn't play nice with dagster